client/highlevel-rest/src/test/java/org/elasticsearch/client/HLClientSearchIT.java

```diff
@@ -45,11 +44,11 @@
 
 public class HLClientSearchIT extends ESRestTestCase {
 
-    private HighlevelClient aClient;
+    private RestHighLevelClient highLevelClient;
```

Comment:
this test is going to be removed together with the search stuff that we did till now, so I am not going to bother renaming the instance member, sorry :)